specifications/transactions/offchain-identity/README.md

```diff
@@ -0,0 +1,96 @@
+# Off-Chain Identity
+
+Diem Payment Network wallets benefit from consistent standards for consistent naming and addressing across participants and their clients.
+
+## Subaddresses
+
+Each account on-chain is represented by a 16-byte value called an **account address**. To allow multiplexing of a single address into distinct off-chain identities, or wallets, a participant may use a **subaddress**. For convenience, Diem defines a standard format for representing the combination of the an account address and a subaddress as an **account identifier**. Account identifiers have a dedicated URI representation including parameters called an **intent identifier**. Subaddresses, account identifiers, and intent identifiers are defined in [DIP-5](https://dip.diem.com/dip-5/).
+
+### Subaddresses
+
+- 8 bytes
+- Unambiguous (i.e., Recipient VASPs should not issue the same subaddress to multiple clients)
+- The zero subaddress (`0x0000000000000000`) is reserved to denote the root (VASP owned) account.
+- Subaddresses should be single-use and should not be generated from personal information, such as names, email addresses, or government-issued identification numbers.
+
+### Account Identifiers
+
+Account Identifier format: `<prefix> | <delimiter> | <version> | <encoded payload> | <checksum>`
+
+- A prefix (also known as hrp (human readable part) which identifies the network version this address is intended for
+  - “dm” for Mainnet addresses
+  - “tdm” for Testnet addresses
+  - "pdm" for Pre-Mainnet addresses
+- A Bech32 delimiter: The character “1” (one)
+- A Bech32 version identifier: The character “p” (version = 1) for on-chain with subaddress
+- A Bech32 encoded payload: For version 1, is Diem account address + subaddress (16 + 8 bytes)
+- The last 6 characters correspond to the Bech32 checksum
+
+The Diem Account Identifier must not be mixed-cases and represented by only all upper or lower case.  For example, both `dm1pptdxvfjck4jyw3rkfnm2mnd2t5qqqqqqqqqqqqq305frg` or `DM1PPTDXVFJCK4JYW3RKFNM2MND2T5QQQQQQQQQQQQQ305FRG` are valid but `dm1pptdXVFJCK4JYW3RKFNM2MND2t5qqqqqqqqqqqqq305frg` is not.
```

Comment:
Is this a standard bech32 thing? Do we have code that actually enforces this?

Reply:
dunno, this was in the DIP :(

I fully intend on killing bech32

Reply:
lead with that man, lead with that ;P 

Anyway fair enough, maybe I should stop nit picking all of these set-in-stone specs lol

Reply:
The first write of these docs was ... a lot more disruptive... and then I realized all it did was make me feel better but was otherwise useless lol...